app/controllers/idv/by_mail/request_letter_controller.rb

```diff
@@ -33,7 +34,7 @@ def self.step_info
           action: :index,
           next_steps: [:enter_password],
           preconditions: ->(idv_session:, user:) do
-            idv_session.verify_info_step_complete? || user.gpo_verification_pending_profile?
+            idv_session.verify_info_step_complete?
```

Comment:
I assume that this is not required here because you're not using the request letter controller any more if you come back in a new session and request a resend?

Reply:
Yeah, this should have been removed in #10866